src/Apps/CollectorProfile/Routes/Saves/Components/ArtworkListArtworksGrid.tsx

```diff
@@ -50,6 +51,11 @@ const ArtworkListArtworksGrid: FC<ArtworkListArtworksGridProps> = ({
   const previousFilters = usePrevious(filters)
 
   const artworks = me.artworkList?.artworks
+
+  const { CollectorSignalsProvider } = useLoadCollectorSignals({
+    artworks,
+  })
+
```

Comment:
Ended up with a load/use with provider hook because artwork details are deeply nested in here. Not 100% set on it though.

Reply:
Leaning more towards the route of having the hook return the mapped `[artworkID]: CollectorSignals` object, and letting other, possibly already existing contexts on the list do the providing if necessary (I would say they are due to the deep nesting of components inside the artwork grid)